packages/plugin-dom-editable/src/DomEditable.ts

```diff
@@ -0,0 +1,202 @@
+import { JWPlugin, JWPluginConfig } from '../../core/src/JWPlugin';
+import { DomLayout } from '../../plugin-dom-layout/src/DomLayout';
+import { Direction } from '../../core/src/VSelection';
+import { EventNormalizer, EventBatch, NormalizedAction } from './EventNormalizer';
+import { CommandIdentifier } from '../../core/src/Dispatcher';
+import { InsertTextParams } from '../../plugin-char/src/Char';
+import { VSelectionParams } from '../../core/src/Core';
+import { Layout } from '../../plugin-layout/src/Layout';
+import { DomComponent, DomLayoutEngine } from '../../plugin-dom-layout/src/ui/DomLayoutEngine';
+import JWEditor, { Loadables } from '../../core/src/JWEditor';
+import { VElement } from '../../core/src/VNodes/VElement';
+import { Renderer } from '../../plugin-renderer/src/Renderer';
+import { EditableDomRenderer } from './EditableDomRenderer';
+import { RelativePosition } from '../../core/src/VNodes/VNode';
+import { Keymap } from '../../plugin-keymap/src/Keymap';
+
+export interface DomEditableConfig extends JWPluginConfig {
+    autoFocus?: boolean;
+}
+
+export class DomEditable<T extends DomEditableConfig = DomEditableConfig> extends JWPlugin<T> {
+    static dependencies = [DomLayout, Layout, Keymap];
+    readonly loadables: Loadables<Renderer & Layout> = {
+        renderers: [EditableDomRenderer],
+        layoutComponents: [],
+    };
+
+    commandHooks = {
+        '*': this._renderInEditable,
+    };
+
+    node: VElement;
```

Comment:
root?

Reply:
It is not necessarily root, I think. Nothing prevents to choose another element.